md/create-a-complex-data-type.md

```diff
@@ -4,12 +4,16 @@
 **Note:** For Bonita Enterprise, Performance, Efficiency, and Teamwork editions only.
 :::
 
+If the standard data types are not suitable for a variable in your process, you can create a new custom complex data type. It could be useful for example to reference data from an external database.
+
 ::: warning
-**Deprecated:** Use a Business Object or a POJO ( using a custom Groovy script or import a jar) instead. POJO must implement _java.io.Serializable_.
+**Deprecated:** 
+Because they are serialized in the database, because they cannot use requests (no API), because they may not work if the Bonita platform Java version is updated, complex data types are now deprecated.
```

Comment:
> because they cannot use requests

Not clear to me what this sentence means...

Reply:
You're right.
Tried another sentence.
Needs validation from @rbioteau